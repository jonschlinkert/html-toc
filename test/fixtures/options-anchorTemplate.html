<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>
<body>
  <div id="toc"></div>
  <h2>Foo</h2>
  <h2>Bar</h2>
  <h2>Baz</h2>
</body>
</html>
