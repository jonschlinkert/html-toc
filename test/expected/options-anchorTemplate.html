<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>
<body>
  <div id="toc"><ul class="nav sidenav"><li><a href="#foo">Foo</a></li>
<li><a href="#bar">Bar</a></li>
<li><a href="#baz">Baz</a></li></ul></div>
  <h2 id="foo">Foo@foo@</h2>
  <h2 id="bar">Bar@bar@</h2>
  <h2 id="baz">Baz@baz@</h2>
</body>
</html>
